kvbc/include/reconfiguration_kvbc_handler.hpp

```diff
@@ -25,9 +25,15 @@ class ReconfigurationBlockTools {
  protected:
   ReconfigurationBlockTools(kvbc::IBlockAdder& block_adder, kvbc::IReader& ro_storage)
       : blocks_adder_{block_adder}, block_metadata_{ro_storage}, ro_storage_{ro_storage} {}
-  kvbc::BlockId persistReconfigurationBlock(const std::vector<uint8_t>& data, const uint64_t bft_seq_num, string key);
+  kvbc::BlockId persistReconfigurationBlock(const std::vector<uint8_t>& data,
```

Comment:
persist wedge block on every wedging command (to have a single mechanism to handle all wedging commands)